### PR TITLE
Added GMK trackday-based theme 

### PR DESCRIPTION
**Pull request by batsoup804**
Here's a few photos!
image
image

### Description

This PR adds the trackday theme. It's based off of the pre-production set GMK trackday, featured on novelkeys.com. I love this set so much I joined the group already but I realized how much I want this in Monkeytype so I decided to add it.
https://novelkeys.com/products/gmk-trackday
feel free to check it out:)

### CODE REVIEW

**Comment by Nugget1308** [comment]
Looks pretty nice, but can you use a brighter red?

File: static/themes/_list.json [hunk 1]
```diff
@@ -669,4 +669,9 @@
      "bgColor" : "#9bf2ea",
      "mainColor":"#6d44a6"
    }
+  ,{
+    "name": "trackday",
+    "bgColor": "#464d66"
```
Comment (rizwanmustafa) [inline static/themes/_list.json:674]: Add a comma at the end of this line.

**Review by rizwanmustafa — CHANGES_REQUESTED** [review]

File: static/themes/_list.json (hunk 1 above)
Comment (batsoup804) [reply static/themes/_list.json:674]: thanks, didnt realize there was a syntax error.

**Comment by batsoup804** [comment]
> Looks pretty nice, but can you use a brighter red?

Well the set itself uses an almost brown so I think it's fine the way it is. Besides the only person's concern about the colors I really care about is Miodec's.

**Review by Miodec — APPROVED** [review]
Thanks

**Pull request merged**